src/components/editor/faultTree/menu/faultEvent/FaultEventMenu.tsx

```diff
@@ -21,7 +22,19 @@ interface Props {
   rootIri?: string;
 }
 
+enum RadioButtonType {
+  Predicted = "Predicted",
+  Manual = "Manual",
+  Operational = "Operational",
+}
+
+enum ManualFailureRateType {
```

Comment:
? this is  strange name of the enum

Reply:
Agree, looks strange. Renamed to "NodeTypeWithManualFailureRate"

Reply:
@blcham For better understanding i'll rename it to "Basic"